CHANGELOG.md

```diff
@@ -6,4 +6,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-[Unreleased]: https://github.com/MetaMask/KeyringController/
+## [7.0.0]
+### Added
+- Add forget Keyring method for some hardware devices ([#124](https://github.com/MetaMask/KeyringController/pull/124))
+- Add `@lavamoat/allow-scripts` ([#109](https://github.com/MetaMask/KeyringController/pull/109))
+
+### Changed
+- **BREAKING**: Bump eth-hd-keyring to latest version ([#132](https://github.com/MetaMask/KeyringController/pull/132))
+    - When calling the `addNewKeyring` method, an options object can no longer be passed containing a `numberOfAccounts` property without also including a `mnemonic`. Not adding any option argument will result in the generation of a new mnemonic and the addition of 1 account derived from that mnemonic to the keyring.
```

Comment:
>Not adding any option argument will result in the generation of a new mnemonic and the addition of 1 account derived from that mnemonic to the keyring.

Wait... what? That doesn't sound right. I thought it would never implicitly generate a mnemonic.

Reply:
Oh sorry, I thought this was referring to HD keyring, I was mixed up.

Reply:
It does raise the question of whether we should be preventing implicit key generation here as well though. That is what the goal was of that change in the keyring.